plugins/woocommerce/includes/wc-formatting-functions.php

```diff
@@ -1272,7 +1272,10 @@ function wc_format_stock_quantity_for_display( $stock_quantity, $product ) {
  * @return string
  */
 function wc_format_sale_price( $regular_price, $sale_price ) {
-	$price = '<del aria-hidden="true">' . ( is_numeric( $regular_price ) ? wc_price( $regular_price ) : $regular_price ) . '</del> <ins>' . ( is_numeric( $sale_price ) ? wc_price( $sale_price ) : $sale_price ) . '</ins>';
+	$price  = '<span class="screen-reader-text">' . __( 'Previous price:', 'woocommerce' ) . '</span>';
+	$price .= '<del>' . ( is_numeric( $regular_price ) ? wc_price( $regular_price ) : $regular_price ) . '</del>';
```

Comment:
Removed `aria-hidden="true"` because we want this to be accessible to screen readers.